--- manedit-1.2.1/manedit/platforms.ini.orig	2014-02-02 11:09:59.282748762 +0100
+++ manedit-1.2.1/manedit/platforms.ini	2014-02-02 11:17:32.332729754 +0100
@@ -23,7 +23,7 @@
 Platform = UNIX
         Description = For most any UNIX system
         PREFIX = /usr/
-        CFLAGS = -Wall -O2 -DNEED_STRCASESTR
+        CFLAGS = -Wall $(OPTFLAGS) -DNEED_STRCASESTR
         LIBS = -lm
         LIB_DIR =
 	CC = cc
@@ -94,7 +94,7 @@
 Platform = Linux
 	Description = For most any Linux system
 	PREFIX = /usr/
-	CFLAGS = -Wall -O2 -DNEED_STRCASESTR
+	CFLAGS = -Wall $(OPTFLAGS)
 	LIBS = -lm
 	LIB_DIR =
         CC = cc
@@ -120,38 +120,6 @@
                 Description = Just adds -g to the CFLAGS for debugging
                 MustExist = No
                 FeatureCFLAGS = -g
-        PlatformFeature = arch-i486
-                Description = Adds -march=i486 for 486 specific optimizations
-                MustExist = Preferred  
-                FeatureCFLAGS = -march=i486
-                FeatureDepend = i486
-                        DependType = Machine
-                        MustExist = Yes
-                        DependMachine = i486
-        PlatformFeature = arch-i586
-                Description = Adds -march=i586 for Pentium specific optimizations
-                MustExist = Preferred
-                FeatureCFLAGS = -march=i586
-		FeatureDepend = i586
-			DependType = Machine
-			MustExist = Yes
-			DependMachine = i586
-        PlatformFeature = arch-i686
-                Description = Adds -march=i686 for Pentium Pro specific optimizations
-                MustExist = Preferred
-                FeatureCFLAGS = -march=i686
-                FeatureDepend = i686
-                        DependType = Machine
-                        MustExist = Yes
-                        DependMachine = i686
-        PlatformFeature = arch-pentiumpro
-                Description = Adds -march=pentiumpro for Pentium Pro specific optimizations
-                MustExist = Preferred
-                FeatureCFLAGS = -march=pentiumpro
-                FeatureDepend = pentiumpro
-                        DependType = Machine
-                        MustExist = Yes
-                        DependMachine = pentiumpro
         PlatformFeature = zlib
                 Description = ZLib
                 MustExist = Preferred
