diff -ur manedit-0.5.6.orig/manedit/platforms.ini manedit-0.5.6/manedit/platforms.ini
--- manedit-0.5.6.orig/manedit/platforms.ini	Tue Mar 12 23:02:37 2002
+++ manedit-0.5.6/manedit/platforms.ini	Sun Mar 24 19:31:42 2002
@@ -69,7 +69,7 @@
                 FeatureDepend = bzip2-lib
                         DependType = Library
                         MustExist = Yes
-                        DependPath = libbz2.so.1.0
+                        DependPath = libbz2.so
                         DependGrepString = BZ2_bzReadOpen
                 FeatureDepend = bzip2-devel
                         DependType = Header
@@ -179,7 +179,7 @@
                 FeatureDepend = bzip2-lib
                         DependType = Library
                         MustExist = Yes
-                        DependPath = libbz2.so.1.0
+                        DependPath = libbz2.so
                         DependGrepString = BZ2_bzReadOpen
                 FeatureDepend = bzip2-devel
                         DependType = Header
